lib/fluent/plugin/out_azure-storage-append-blob.rb

```diff
@@ -65,7 +66,17 @@ def multi_workers_ready?
       def start
         super
   
-        @bs = Azure::Storage::Blob::BlobService.create(storage_account_name: @azure_storage_account, storage_access_key: @azure_storage_access_key)
+        @bs_params = {storage_account_name: @azure_storage_account}
+
+        if !@azure_storage_access_key.nil?
```

Comment:
Can you please add validation that at least one parameter is provided? (Similar to the check in another PR)

Reply:
Sure, done in e30393a132460fcc1ba1f2fe968c071ce65aedd4, thank you!

Reply:
I think you want to be using `&&` instead of `||`, right?